Longest Common Subsequence

Bhaswati-Saha opened the pull request:
Longest Common Subsequence

Fixes: #3 

Made modifications. Given proper class name and comments. Removed extra lines and made a simple table format for contribution. Also I have edited the title of the pull request.

Type of change
[ Y] New feature (non-breaking change which adds functionality)
Checklist:
[ Y] My code follows the style guidelines of this project
[ Y] I have performed a self-review of my own code
[ Y] I have commented my code, particularly in hard-to-understand areas
[ Y] I have made corresponding changes to the documentation
[ Y] My changes generates no new warnings
[ Y] I have added tests that prove my fix is effective or that my feature works
[ Y] New and existing unit tests pass locally with my changes
[ Y] Any dependent changes have been merged and published in downstream modules

Bhaswati-Saha committed:
Here is my Java solution for diamond pattern

Bhaswati-Saha committed:
Here is my Java solution for longest common subsequence

Bhaswati-Saha committed:
Here is my java solution for Longest Common Subsequence

kaustubhgupta commented on snippets/java/LongestCommonSubsequence.md:
```diff
@@ -0,0 +1,91 @@
+# Longest Common Subsequence
```
Edit the title of this pull request to Longest Common Subsequence

sakshamarora1 replied:
And update the PR message accordingly too

kaustubhgupta commented on snippets/java/LongestCommonSubsequence.md:
````diff
@@ -0,0 +1,91 @@
+```
+
+## Contributed By
+-----------------------------------------------------------------------------------------------------------------
````
Remove extra lines from this and follow the simple table format

kaustubhgupta commented on snippets/java/diamondSquare.md:
```diff
@@ -0,0 +1,65 @@
+# Print a Diamond pattern enclosed in a square
```
Remove this file from this pull request. Wait, didn't you make separate branches for separate issues? If not then kindly do that else clean up the extra file in this  pull request

Bhaswati-Saha replied:
Ok will do it .Thanks for the review.

sakshamarora1 commented on snippets/java/LongestCommonSubsequence.md:
```diff
@@ -0,0 +1,91 @@
+            {
+                if(i==0 || j==0)
+                    t[i][j]=0;
+
```
```suggestion
```

sakshamarora1 commented on snippets/java/LongestCommonSubsequence.md:
````diff
@@ -0,0 +1,91 @@
+## Code
+```Java
+import java.util.*;
+public class LCS3 {
````
Provide better class names and better comments with explanation for a clearer understanding for others too.

sakshamarora1 commented on snippets/java/LongestCommonSubsequence.md:
```diff
@@ -0,0 +1,91 @@
+                s=X.charAt(i-1)+s;
+                i--;
+                j--;
+
```
```suggestion
```

Bhaswati-Saha committed:
Made the modifications.Given a proper class name and comments.Removed extra lines and form a simple table format for contribution

Bhaswati-Saha commented:
Longest Common Subsequence 

Related: Issue #3 

Made modifications. Given proper class name and comments. Removed extra lines and made a simple table format for contribution. Also I have edited the title of the pull request.

Type of change
[ Y] New feature (non-breaking change which adds functionality)
Checklist:
[ Y] My code follows the style guidelines of this project
[ Y] I have performed a self-review of my own code
[ Y] I have commented my code, particularly in hard-to-understand areas
[ Y] I have made corresponding changes to the documentation
[ Y] My changes generates no new warnings
[ Y] I have added tests that prove my fix is effective or that my feature works
[ Y] New and existing unit tests pass locally with my changes
[ Y] Any dependent changes have been merged and published in downstream modules

Bhaswati-Saha committed:
Delete desktop.ini

kaustubhgupta commented:
Edit the PR template instead of posting it as a comment

Bhaswati-Saha commented:
> Edit the PR template instead of posting it as a comment

Means in the previous PR message I will just have to specify the changes I have made in the "Type of change" field and I don't have to comment any further?

Bhaswati-Saha commented:
> Edit the PR template instead of posting it as a comment

okay I have edited the PR template. Please check it if there is any more errors.

sakshamarora1 commented on snippets/java/LongestCommonSubsequence.md:
````diff
@@ -0,0 +1,93 @@
+}
+```
+
+## Description
````
It will be better if you shift the description above the code sub-heading.

Bhaswati-Saha replied:
Done

sakshamarora1 commented on snippets/java/LongestCommonSubsequence.md:
````diff
@@ -0,0 +1,93 @@
+```
+
+## Description
+`Scanner` is a java.util package class that is used to take input from console. 
````
Add a little more description on how the program works in 2-3 lines too.

Bhaswati-Saha replied:
Done

Bhaswati-Saha committed:
Modified description block and moved it above the code block

kaustubhgupta reviewed (approved):
LGTM :-)
